main.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 (function () {
-  const clientId = '60b252599fc55fd7f79c91370e37e141408cf38df7e0fa129130afd960bb683e';
+  const clientId = '07592e512632cbce9dc6a8ff5064296c78ef1c2ba8b88b2004d1429ed6fbffed';
```

Comment:
What's the reason for changing the client ID? Does this need to match a URL set up with YNAB? I can update my ID to support the GitHub.io url if desired. I think it's currently set up for https://cdn.rawgit.com/alanrussian/ynab-transaction-resurrector/69f7f489/index.html